cli_tools/common/disk/inspect_test.go

```diff
@@ -35,13 +34,12 @@ import (
 
 func TestBootInspector_Inspect_PassesVarsWhenInvokingWorkflow(t *testing.T) {
 	for caseNumber, tt := range []struct {
-		inspectOS bool
 		reference string
 	}{
-		{inspectOS: true, reference: "uri/for/pd"},
-		{inspectOS: false, reference: "uri/for/pd"},
+		{reference: "uri/for/pd"},
```

Comment:
then it's one single test case, no need to loop

Reply:
done!